packages/dds/tree/src/test/feature-libraries/default-schema/schemaChecker.spec.ts

```diff
@@ -490,4 +497,22 @@ describe("schema validation", () => {
 			);
 		});
 	});
+
+	describe("testTrees", () => {
```

Comment:
When updating test trees, I had a bug which caused them to be out of schema, and nothing detected that, so I added these tests and confirmed it caught the issue.